### DISCUSSION TITLE
Feature request: Maximising chat on full window

### DISCUSSION THREAD

**Discussion by ThBa** [Chatterino/chatterino2#4354] [Ideas]
On a tab with several chats open, would it be possible to have a shortcut to expand the active chat to the whole window? Reusing this shortcut will bring it back to its previous place.

The behavior I'm imagining is exactly that of Terminator with one active shell among several visible ones and the keyboard shortcut `Ctrl`+`Shift`+`X` to zoom in/out on one ([doc here](https://gnome-terminator.readthedocs.io/en/latest/gettingstarted.html#using-the-keyboard), [example gif](https://i.imgur.com/HGw0J1o.gif)).

**Comment by Mm2PL** [comment]
This sounds niche. I'd be down to introduce lua functionality that allows for this in the future though if/when #4341 gets merged.
